base/src/test/java/uk/ac/ebi/atlas/bioentity/interpro/InterProTSVParserTest.java

```diff
@@ -1,81 +1,69 @@
 package uk.ac.ebi.atlas.bioentity.interpro;
 
-import au.com.bytecode.opencsv.CSVReader;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import uk.ac.ebi.atlas.commons.readers.TsvStreamer;
 import uk.ac.ebi.atlas.model.OntologyTerm;
 
-import java.io.IOException;
 import java.util.Map;
+import java.util.stream.Stream;
 
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-import static org.mockito.BDDMockito.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
-public class InterProTSVParserTest {
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class InterProTSVParserTest {
     private static final String IPR000001 = "IPR000001";
     private static final String KRINGLE = "Kringle";
```

Comment:
Apparently a Kringle is a type of Scandinavian pastry! 😄 Methinks a biologist was hungry when they came up with this particular concept.